### PR TITLE
Add OWNERS file, Fix #753

### LINKED ISSUE #753: This Repository Is Missing an OWNERS File
**What keywords did you search in Kubernetes issues before filing this one?** (If you have found any duplicates, you should instead reply there.): OWNER, OWNERS, owners

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): FEATURE REQUEST

**Description**
Most other kubernetes repos have a OWNERS file that lists approvers, reviewers, etc.  This is helpful when you need to ping someone to get an issue or PR moving.

Please create an OWNERS file for this repository!

### PR DESCRIPTION
**Pull request by pierreozoux**

### CODE REVIEW

**Comment by lachie83** [comment]
cc @prydonius @viglesiasce

**Comment by prydonius** [comment]
I'm fine with this, although I wonder if we want to add some reviewers?

**Comment by lachie83** [comment]
Do you have a list you can add to this PR?

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+maintainers:
```
Comment (prydonius) [inline OWNERS:1]: What's the correct syntax for the OWNERS file? In https://github.com/kubernetes/kubernetes/blob/master/OWNERS there are approvers, and in https://github.com/kubernetes/helm/blob/master/OWNERS there are maintainers. In other repos (e.g. kops), they only have assignees.

**Comment by lachie83** [comment]
Discussed. We're not going to add reviewers just yet. Following helm repo owners file structure for now.

**Pull request merged**